code/modules/admin/admin_verbs.dm

```diff
@@ -109,138 +109,141 @@ GLOBAL_LIST_INIT(admin_verbs_server, world.AVerbsServer())
-	/client/proc/cinematic,
-	/client/proc/cmd_admin_add_freeform_ai_law,
-	/client/proc/cmd_admin_create_centcom_report,
-	/client/proc/cmd_change_command_name,
-	/client/proc/object_say,
-	/client/proc/toggle_random_events,
+	/datum/tgclient/proc/admin_call_shuttle,
+	/datum/tgclient/proc/admin_cancel_shuttle,
+	/datum/tgclient/proc/cmd_admin_direct_narrate,
+	/datum/tgclient/proc/cmd_admin_world_narrate,
+	/datum/tgclient/proc/cmd_admin_local_narrate,
+	/datum/tgclient/proc/play_local_sound,
+	/datum/tgclient/proc/play_sound,
+	/datum/tgclient/proc/set_round_end_sound,
+	/datum/tgclient/proc/cmd_admin_dress,
+	/datum/tgclient/proc/cmd_admin_gib_self,
+	/datum/tgclient/proc/drop_bomb,
+	/datum/tgclient/proc/drop_dynex_bomb,
+	/datum/tgclient/proc/get_dynex_range,
+	/datum/tgclient/proc/get_dynex_power,
+	/datum/tgclient/proc/set_dynex_scale,
+	/datum/tgclient/proc/cinematic,
+	/datum/tgclient/proc/cmd_admin_add_freeform_ai_law,
+	/datum/tgclient/proc/cmd_admin_create_centcom_report,
+	/datum/tgclient/proc/cmd_change_command_name,
+	/datum/tgclient/proc/object_say,
+	/datum/tgclient/proc/toggle_random_events,
 	/datum/admins/proc/startnow,
 	/datum/admins/proc/restart,
 	/datum/admins/proc/delay,
 	/datum/admins/proc/toggleaban,
-	/client/proc/everyone_random,
+	/datum/tgclient/proc/everyone_random,
 	/datum/admins/proc/toggleAI,
-	/client/proc/restart_controller,
-	/client/proc/cmd_admin_list_open_jobs,
-	/client/proc/callproc,
-	/client/proc/callproc_datum,
-	/client/proc/Debug2,
-	/client/proc/reload_admins,
-	/client/proc/cmd_debug_make_powernets,
-	/client/proc/startSinglo,
-	/client/proc/cmd_debug_mob_lists,
-	/client/proc/cmd_debug_del_all,
-	/client/proc/enable_debug_verbs,
+	/datum/tgclient/proc/restart_controller,
+	/datum/tgclient/proc/cmd_admin_list_open_jobs,
+	/datum/tgclient/proc/callproc,
+	/datum/tgclient/proc/callproc_datum,
+	/datum/tgclient/proc/Debug2,
+	/datum/tgclient/proc/reload_admins,
+	/datum/tgclient/proc/cmd_debug_make_powernets,
+	/datum/tgclient/proc/startSinglo,
+	/datum/tgclient/proc/cmd_debug_mob_lists,
+	/datum/tgclient/proc/cmd_debug_del_all,
+	/datum/tgclient/proc/enable_debug_verbs,
 	/proc/possess,
 	/proc/release,
-	/client/proc/reload_admins,
-	/client/proc/panicbunker,
-	/client/proc/admin_change_sec_level,
-	/client/proc/toggle_nuke,
-	/client/proc/cmd_display_del_log,
-	/client/proc/toggle_antag_hud,
-	/client/proc/debug_huds,
-	/client/proc/customiseSNPC,
-	/client/proc/resetSNPC,
+	/datum/tgclient/proc/reload_admins,
+	/datum/tgclient/proc/panicbunker,
+	/datum/tgclient/proc/admin_change_sec_level,
+	/datum/tgclient/proc/toggle_nuke,
+	/datum/tgclient/proc/cmd_display_del_log,
+	/datum/tgclient/proc/toggle_antag_hud,
+	/datum/tgclient/proc/debug_huds,
+	/datum/tgclient/proc/customiseSNPC,
+	/datum/tgclient/proc/resetSNPC,
 	))
 
-/client/proc/add_admin_verbs()
+/client/add_admin_verbs()
 	if(holder)
 		control_freak = CONTROL_FREAK_SKIN | CONTROL_FREAK_MACROS
+	return ..()
```

Comment:
This one proc is an oddity because overriding vars which you set an compile time can be weird